quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliVersionIT.java

```diff
@@ -24,11 +38,35 @@
     static QuarkusCliClient cliClient;
 
     @Test
+    @DisabledIfSystemProperty(named = "quarkus.platform.group-id", matches = "com.redhat.quarkus.platform", disabledReason = "productized CLI doesn't exist yet")
```

Comment:
Please add a  `//TODO` statement in order to track this disabled test.
Example:

//TODO Enable test once Quarkus Cli is released on the product side.

Reply:
There are no plans to have productized CLI, community one is planned to be used against productized bits and product registry